app/views/layouts/mailer.text.erb

```diff
@@ -10,7 +10,7 @@
 <% end %>
 
 <% yield_with_default_for :fine_print do %>
-  High School Hackathons is maintained by Hack Club (https://hackclub.com) staff.
+  <%= link_to "hackathons.hackclub.com", "https://hackathons.hackclub.com" %> is maintained by Hack Club (<%= link_to "hackclub.com", "https://hackclub.com" %>) staff.
```

Comment:
Using `link_to` in the text version results in HTML
![image](https://github.com/hackclub/hackathons-backend/assets/20099646/0dc0f906-4281-43dd-a2c3-1f96f2eaae36)

Instead, we should manually render the link in parentheses.